src/bot.ts

```diff
@@ -38,13 +45,40 @@ export const startBot = async (): Promise<void> => {
     logger.info({
       event: 'init'
     });
-    const notif = (await client.channels.fetch(NOTIF_CHANNEL_ID)) as Discord.TextChannel;
+
+    const notif = (await client.channels.fetch(NOTIF_CHANNEL_ID)) as TextChannel;
+
     initEmojis(client);
     createSuggestionCron(client).start();
+
+    // Intialize Bootcamp features if the event server exists
```

Comment:
could we put this section in its own function?

Reply:
yeah, i think we could just move all that logic into initBootcamp